pom.xml

```diff
@@ -883,7 +883,6 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-release-plugin</artifactId>
-                    <version>${cs.release-plugin.version}</version>
```

Comment:
also inherited from parent

Reply:
I'm not sure if root pom.xml has a parent cc @DaanHoogland